packages/bitcore-wallet-service/src/lib/chain/btc/index.ts

```diff
@@ -373,7 +373,7 @@ export class BtcChain implements IChain {
 
   addressToStorageTransform(network, address) {}
 
-  addSignaturesToBitcoreTx(tx, inputs, inputPaths, signatures, xpub) {
+  addSignaturesToBitcoreTx(tx, inputs, inputPaths, signatures, xpub, signingMethod) {
     if (signatures.length != inputs.length) throw new Error('Number of signatures does not match number of inputs');
```

Comment:
given this is btc, maybe add:

$.checkState(!signedMethod, "signedMethod should not be given for BTC").